addons/l10n_ir_accounting/data/account_fiscal_position_data.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data noupdate="1">
+        <!-- Fiscal Position Templates -->
+        <record
+            id="fiscal_position_template_1"
```

Comment:
fiscal template and fiscal position are usually in two different files. Also the indentation is not consistent between the two records